fuel-txpool/src/txpool.rs

```diff
@@ -239,10 +241,16 @@ impl TxPool {
 
     /// When block is updated we need to receive all spend outputs and remove them from txpool.
     pub async fn block_update(
-        txpool: &RwLock<Self>, // spend_outputs: [Input], added_outputs: [AddedOutputs]
+        txpool: &RwLock<Self>,
+        block: Arc<FuelBlock>,
+        // spend_outputs: [Input], added_outputs: [AddedOutputs]
     ) {
-        let _ = txpool.write().await;
+        let mut guard = txpool.write().await;
         // TODO https://github.com/FuelLabs/fuel-core/issues/465
```

Comment:
Should this still be TODO if this PR resolves that issue?

Reply:
The issue is only partially resolved here, and the TODO should indeed remain.